setup/updating.md

```diff
@@ -44,13 +44,15 @@ To update Coder, follow these steps:
     helm get values --namespace coder coder > current-values.yml
     ```
 
+    > Double-check your values to ensure it only contains your changes.  
+
 1. Provide your helm chart values file and upgrade to the desired version (e.g.,
 1.16.1):
 
     *Note: If you omit --version, you'll upgrade to the latest version.*
 
     ```bash
-    helm upgrade --namespace coder --force --install --atomic --wait \
+    helm upgrade --namespace coder --install --atomic --wait \
```

Comment:
@sreya said the --force flag causes the helm errors, my install succeeded when I removed it.